Sources/AppStoreConnectCLI/Commands/TestFlight/PreReleaseVersion/ListPreReleaseVersionsCommand.swift

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Foundation
+
+struct ListPreReleaseVersionsCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "list",
+        abstract: "Get a list of prerelease versions for all apps.")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Option(
+        parsing: .upToNextOption,
+        help: ArgumentHelp(
+            "Filter by app bundle identifier. eg. com.example.App",
+            discussion: "This option is mutually exclusive with --filter-app-id.",
+            valueName: "bundle-id"
+        )
+    )
+    var filterBundleIds: [String]
+
+    @Option(
+        parsing: .upToNextOption,
+        help: ArgumentHelp(
+            "Filter by app AppStore ID. eg. 432156789",
+            discussion: "This option is mutually exclusive with --filter-bundle-id.",
+            valueName: "app-id"
+        )
+    )
+    var filterAppIds: [String]
```

Comment:
In ReadAppCommand we have something like this, can you try to reuse it? (eg. may be move this to .AppStoreConnectCLI/Arguments) and use an `@OptionGroup`
```suggestion
@Option(
        help: ArgumentHelp(
            "The app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
            discussion: "Please input either app id or bundle Id",
            valueName: "app-id / bundle-id"
        ),
        transform: Identifier.init
    ) var identifier: Identifier

    enum Identifier {
        case appId(String)
        case bundleId(String)

        init(_ argument: String) {
            switch Int(argument) == nil {
            case true:
                self = .bundleId(argument)
            case false:
                self = .appId(argument)
            }
        }
    }
```

Reply:
Creating an option group for this seems like a good idea since we use it quite a bit

Reply:
I need to use the identifier as an option in list command and as an argument in read command. So will one option group work?

Reply:
Might need two in that case, but we are using them for other commands so we should still get some reuse

Reply:
well, list command should be able to take an array of identifiers..so we may need three

Reply:
Maybe we should have a separate PR for refactoring this..at the moment maybe I should keep it in individual files..what's your thought on this?